src/runtime/web/index.js

```diff
@@ -40,9 +40,14 @@ function init() {
   this.views.push(
     document.l10n = new View(this, document));
 
+  var prevLangs = [];
+  if (document.documentElement.hasAttribute('lang')) {
```

Comment:
Any logic directly related to the `document` should probably live in the view, not here.